src/language/__tests__/schema-parser-test.js

```diff
@@ -74,99 +75,102 @@ function inputValueNode(name, type, defaultValue, loc) {
 
 describe('Schema Parser', () => {
   it('Simple type', () => {
-    const doc = parse(`
-type Hello {
-  world: String
-}`);
+    const doc = parse(dedent`
+      type Hello {
+        world: String
+      }
+    `);
 
     expect(toJSONDeep(doc)).to.deep.equal({
       kind: 'Document',
       definitions: [
         {
           kind: 'ObjectTypeDefinition',
-          name: nameNode('Hello', { start: 6, end: 11 }),
+          name: nameNode('Hello', { start: 5, end: 10 }),
```

Comment:
All offsets are shifted by 1 because template literal contained initial new line after \` but it's striped by `dedent`.